### PR TITLE
Adding files for Windows Server BYOI for HPE Private Cloud Enterprise - Bare Metal

### PR DESCRIPTION
**Pull request by nareshbhalala**
Adding files for **_Microsoft Windows Server Bring Your Own Image (BYOI) for HPE Private Cloud Enterprise - Bare Metal_**
```
./LICENSE
./CONTRIBUTING.md
./README.md
./.gitignore
./*.xml
./*.dos
./*.ps1
Config/*.ps1
Modules/*.ps1
```

### CODE REVIEW

File: Autounattend-2019.xml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                        </ModifyPartition>
+                    </ModifyPartitions>
+                </Disk>
+            </DiskConfiguration>
+            <ImageInstall>
+                <OSImage>
+                    <InstallFrom>
+                        <MetaData wcm:action="add">
+				            <Key>/IMAGE/INDEX</Key>
+                            <Value>INSTALLINDEX</Value>
```
Comment (neilgierman) [inline Autounattend-2019.xml]:
```suggestion
                        <MetaData wcm:action="add">
                            <Key>/IMAGE/INDEX</Key>
                            <Value>INSTALLINDEX</Value>
```

File: Autounattend-2022.xml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                        </ModifyPartition>
+                    </ModifyPartitions>
+                </Disk>
+            </DiskConfiguration>
+            <ImageInstall>
+                <OSImage>
+                    <InstallFrom>
+                        <MetaData wcm:action="add">
+				            <Key>/IMAGE/INDEX</Key>
+                            <Value>INSTALLINDEX</Value>
```
Comment (neilgierman) [inline Autounattend-2022.xml]:
```suggestion
                        <MetaData wcm:action="add">
                            <Key>/IMAGE/INDEX</Key>
                            <Value>INSTALLINDEX</Value>
```

**Review by neilgierman — COMMENTED** [review]
2 minor formatting issues. After this, we should add branch protection rules on the master branch.

**Review by neilgierman — APPROVED** [review]

**Review by neilgierman — APPROVED** [review]

**Comment by nareshbhalala** [comment]
> 2 minor formatting issues. After this, we should add branch protection rules on the master branch.

1. "2 minor formatting issues" related "commit suggestions" added.
2. The branch protection rules on the master branch is also added.

**Pull request merged**